likeHack.js

```diff
@@ -0,0 +1,111 @@
+import mongoose from "mongoose";
+
+const uri = `mongodb://${process.env.MONGO_USER}:${process.env.MONGO_PASSWORD}@ds157136.mlab.com:57136/hackone`;
+let conn = null;
+
+export const like = async (event, context) => {
```

Comment:
I'd have different functions, one for like one for unlike. based on whether the user has liked it, change the lambda you call from the front end.

You can use mongodb's [`addToSet`](https://docs.mongodb.com/manual/reference/operator/update/addToSet/) operation to stop duplicate user codes going into the array.

This will be cleaner code as you wont need to lookup the hack first, just update it, and makes the actions idempotent (you can retry them safely without fear of the system ending up in an inconsistent state compared to the users intent).